roundtripper.go

```diff
@@ -64,7 +64,7 @@ func (ctx *ProxyCtx) _roundTripWithLog(req *http.Request) (*http.Response, error
 		reqAndResp.captureContent = ctx.isLogWithContent
 
 		req := ctx.Req
-		if reqAndResp.captureContent && req.ContentLength > 0 {
```

Comment:
You also are doing this here https://github.com/miguel250/goproxy/blob/master/har/har.go#L269. The user won't get the content. The issue is that if you don't copy the response `Bodysize` is empty which ` reqAndResp.captureContent && req.ContentLength > 0` is doing.